src/components/accession2/Accession2Create.tsx

```diff
@@ -67,28 +80,48 @@ export default function CreateAccession(props: CreateAccessionProps): JSX.Elemen
     ...marginTop,
   };
 
+  const setDateError = (id: string, error?: string) => {
+    setDateErrors((prev) => ({
+      ...prev,
+      [id]: error,
+    }));
+  };
+
   const changeDate = (id: string, value?: any) => {
     setDates((curr) => ({ ...curr, [id]: value }));
     const date = new Date(value).getTime();
     const now = Date.now();
-    if (isNaN(date) || date > now) {
+
+    setDateError(id, '');
+
+    if (isNaN(date)) {
+      setDateError(id, strings.INVALID_DATE);
+      return;
+    } else if (date > now) {
```

Comment:
Not using the server's time here.. but can make that change if needed.

Reply:
I don't think is necessary